input/fsh/UZCoreAllergyIntolerance.fsh

```diff
@@ -0,0 +1,76 @@
+Profile: UZCoreAllergyIntolerance
+Parent: AllergyIntolerance
```

Comment:
Profiles for AllergyIntolerance, EpisodeOfCare, and Questionnaire shouldn't be in this branch.

When starting a new profile, make a new branch from the `main` branch in this repository, not the last branch you worked with.